src/Iris/Colour/Formatting.hs

```diff
@@ -80,3 +84,57 @@ putStderrColouredLn formatWithColour str = do
     liftIO $ BS8.hPutStrLn stderr $ case colourMode of
         DisableColour -> str
         EnableColour  -> formatWithColour str
+
+{- | Print 'Text' to 'System.IO.stdout' by providing a custom
+formatting function. Doesn't breaks output line that differs from
+`putStdoutColouredLn`
+
+This works especially well with the @colourista@ package:
+
+@
+'putStdoutColoured'
+    (Colourista.formatWith [Colourista.bold, Colourista.green])
+    "my message"
+@
+
+@since 0.0.0.0
```

Comment:
Let's specify the `x.x.x.x` version for all new functions in types. I'll run a command to update all versions before the next release when the actual release version is known

```suggestion
@since x.x.x.x
```